pages/index.tsx

```diff
@@ -17,6 +17,7 @@ export interface DelegateData {
   spendableBalance: number
   stakedBalance: number
   unstakedBalance: number
+  delegate: string | null
```

Comment:
1. can we rename this to `bakerAddress`? for better naming
2. i think it shoule be bakerAddress?: string

Reply:
Removed it as we did not need it.